test/environment_test.exs

```diff
@@ -0,0 +1,15 @@
+defmodule Honeybadger.EnvironmentTest do
```

Comment:
Whoops. These are for my branch for extracting the environment setup to a module. I'll remove from here :)